docs/docsite/rst/network/user_guide/network_debug_troubleshooting.rst

```diff
@@ -303,7 +303,7 @@ Indicates that the remote host you are trying to connect to can not be reached
 
 For example:
 
-.. code-block:: yaml
+.. code-block:: text
```

Comment:
Honestly, I'd mark logs as `console`
```suggestion
.. code-block:: console
```
WDYT?

Reply:
I'll give it a shot but think there was an issue with the console lexer before.